flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -2561,6 +2561,9 @@
     "com.hack_computer.OperatingSystemApp": {
         "appstream-id-mismatch-flatpak-id": "appstream-cid predates this linter rule"
     },
+    "dev.overlayed.Overlayed": {
+        "flathub-json-automerge-enabled": "Predates the linter rule"
```

Comment:
Can you add an actual reason here? "Predates .." is used for the ones we grandfathered in.